config/routes.rb

```diff
@@ -12,6 +12,9 @@
   get "users/:author_id/posts/:id/comments/new", to: "comments#new"
   post "users/:author_id/posts/:id/comments", to: "comments#create"
   post "users/:author_id/posts/:id", to: "likes#create", as: "like_post"
+  get "api/users/:author_id/posts", to: "api/posts#index"
+  get "api/users/:author_id/posts/:id", to: "api/comments#index"
+  post "api/users/:author_id/posts/:id/comments/new", to: "api/comments#create"
```

Comment:
- [ ] Please make sure that your API endpoint that adds' comment to a post - **POST :** `/users/:user_id/posts/:id/comments` :pray: 
- [ ] I suggest that your app has API endpoint that returns all comments for a post - **GET :** `/users/:user_id/posts/:id/comments`. If I get this `/users/:user_id/posts/:id/comments` route I get this error :point_down: 

![image](https://github.com/Tshobohwa/webwisesage/assets/73854999/9680e184-a9b6-4ca7-b14c-f8debca18c6f)